sdk/communication/Azure.Communication.Sms/tests/SmsClientRecordedTestSanitizer.cs

```diff
@@ -7,13 +7,10 @@ namespace Azure.Communication.Sms.Tests
 {
     public class SmsClientRecordedTestSanitizer : CommunicationRecordedTestSanitizer
     {
-        public SmsClientRecordedTestSanitizer() : base()
+        public SmsClientRecordedTestSanitizer()
         {
-            JsonPathSanitizers.Add("$..token");
-            JsonPathSanitizers.Add("$..id");
             JsonPathSanitizers.Add("$..from");
             JsonPathSanitizers.Add("$..to");
-            JsonPathSanitizers.Add("$..messageId");
```

Comment:
Guess we didn't really needed to sanitize what you just removed?

Should we re-record the Sms tests too?

Reply:
Right now, we cannot record everything. There are some server issues

Reply:
We can rerecord them later. In fact @ankitarorabit has to do them with all the new changes in the swagger anyways